test/time.test.ts

```diff
@@ -67,5 +67,56 @@ describe("time helpers", function () {
         expect(result).to.eq(expectedBlockNumber)
       })
     })
+
+    describe("mineBlocksTo function", function () {
+      const blocksToMine = 20
+      let targetBlock: number
+      let result: number
+
+      beforeEach(async function () {
+        const currentBlockNumber = (
+          await this.hre.ethers.provider.getBlock("latest")
+        ).number
+        targetBlock = currentBlockNumber + blocksToMine
+
+        result = await timeHelpers.mineBlocksTo(targetBlock)
+      })
+
+      it("should mine blocks", async function () {
+        expect(
+          (await this.hre.ethers.provider.getBlock("latest")).number
+        ).to.be.eq(targetBlock)
+      })
+
+      it("should return increased block number", async function () {
+        const currentBlockNumber = (
+          await this.hre.ethers.provider.getBlock("latest")
+        ).number
+
+        const targetBlock = currentBlockNumber
+
+        await timeHelpers.mineBlocksTo(targetBlock)
+
+        expect(result).to.eq(targetBlock)
+      })
+
+      it("should complete if target block number is the latest block number", async function () {
+        expect(
+          (await this.hre.ethers.provider.getBlock("latest")).number
+        ).to.be.eq(targetBlock)
+      })
```

Comment:
I am not following this test. Should it call `mineBlocksTo` again with the latest block provided as a parameter?

Reply:
I simplified the tests in d4da3c21483f6d9f9648d460e0de80e069635c32